### PR TITLE
The ESS CSC was split into three separate packages, and as such addit…

### PR DESCRIPTION
**Pull request by rbovill**
…ional Email and Slack notifications entries were added to Csc.groovy.

### CODE REVIEW

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 1]
```diff
@@ -16,6 +16,8 @@ def email() {
                 "ts-eas": "wvreeven@lsst.org",
                 "ts-electrometer": "ecoughlin@lsst.org",
                 "ts-ess": "wvreeven@lsst.org",
```
Comment (wvreeven) [inline src/org/lsst/ts/jenkins/components/Csc.groovy:18]: This should be `ts-ess-csc` now.

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 2]
```diff
@@ -68,6 +70,8 @@ def slack_id() {
                 "ts-eas": wouter,
                 "ts-electrometer": eric,
                 "ts-ess": wouter,
```
Comment (wvreeven) [inline src/org/lsst/ts/jenkins/components/Csc.groovy:72]: This should be `ts-ess-csc` now.

**Review by wvreeven — CHANGES_REQUESTED** [review]
Please fix the `ts-ess` references.

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 1 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy:18]: Fixed

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 2 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy:72]: Fixed

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 1 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy:18]: Unfixed

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 2 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy:72]: Unfixed

**Review by wvreeven — APPROVED** [review]
LGTM

**Pull request merged**